### PR TITLE
doc(wildfly): document WildFly to Tomcat conversion

### PR DESCRIPTION
**Pull request by educhastenier**
Closes [BR-131](https://bonitasoft.atlassian.net/browse/BR-131)

### CODE REVIEW

**Comment by baptistemesta** [comment]
LGTM.
Only one question, should we put that in the taxonomy in addition to the link in the release note?

**Comment by educhastenier** [comment]
> LGTM.
> Only one question, should we put that in the taxonomy in addition to the link in the release note?

Good question. Would be a good idea. But where in the tree?

**Comment by educhastenier** [comment]
Ok, I put the page in the INSTALLATION section

File: md/convert-wildfly-into-tomcat.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* update your license file by putting the new one into `setup/platform_conf/licenses` (and by removing any old license file)
+* run `setup/setup.sh push` to update the new license file (and optional changed configuration) to the database
+* start your new Bonita Tomcat bundle with `./start-bonita.sh`
+* (delete your old WildFly installation by removing the Bonita WildFly root folder)
+
+## Specific configuration
+If you are using specific configuration that you set up in file `wildfly/server/standalone/configuration/standalone.xml`,
+report you equivalent configuration into Tomcat.  
+For instance, if you use Bonita datasource connector, and your configured it to access to a datasource defined in WildFly
+`standalon.xml` file, report your datasource configuration in Tomcat file `server/conf/Catalina/localhost/bonita.xml`.
```
Comment (pabloalonso) [inline md/convert-wildfly-into-tomcat.md:23]: `standalon.xml`  should be `standalone.xml`

**Pull request merged**